### PR TITLE
add integration tab extension for wott

### LINKED ISSUE #1155: Add "Integration tab" extension for WoTT
We need to add a new section to the "Integration tab" if WoTT is detected and installed and populate it with the "Device ID" from WoTT.

### PR DESCRIPTION
**Pull request by eugenosm**
closes #1155

### CODE REVIEW

File: lib/utils.py [hunk 1]
```diff
@@ -329,3 +331,30 @@ def is_balena_app():
     :return: bool
     """
     return bool(getenv('RESIN', False)) or bool(getenv('BALENA', False))
+
+
+def is_wott_integrated():
```
Comment (vpetersson) [inline lib/utils.py]: @eugenosm this won't work well across runtime environments (such as Docker). It's also very hackish. Just do if `WOTT_PATH` exists, then assume the agent is installed.

File: lib/utils.py [hunk 2]
```diff
@@ -329,3 +331,30 @@ def is_balena_app():
...
+    """
+    :return: WoTT Device id of this device
+    """
+    metadata_path = os.path.join(WOTT_PATH, 'metadata.json')
+    if os.path.isfile(metadata_path):
+        with open(metadata_path) as metadata_file:
+            metadata = json.load(metadata_file)
+        if 'device_id' in metadata:
+            return metadata['device_id']
+    return '-- can not read device id --'
```
Comment (vpetersson) [inline lib/utils.py]: This should be logged, not just returned. Also, should be "Could not read WoTT Device ID".

File: templates/settings.html [hunk 3]
```diff
@@ -174,48 +174,31 @@ <h1 class="page-header">
...
                         </div>
                     </div>
-                    <div class="form-group row" id="user_group">
-                        <label class="col-2 col-form-label">User</label>
+                    {% if context.need_current_password %}
+                    <div class="form-group row" id="curpassword_group">
+                        <label class="col-2 col-form-label">Current Password</label>
                         <div class="col-3">
-                            <input class="form-control" name="user" type="text" value="{{ context.user }}">
+                            <input class="form-control" name="curpassword" type="password" value="">
```
Comment (vpetersson) [inline templates/settings.html]: "curpassword" -> "current-password"

**Comment by vpetersson** [comment]
@a-martynovich could you please review this one?

File: lib/utils.py (hunk 1 above)
Comment (eugenosm) [reply lib/utils.py]: ok

File: lib/utils.py (hunk 2 above)
Comment (eugenosm) [reply lib/utils.py]: fixed

File: templates/settings.html (hunk 3 above)
Comment (eugenosm) [reply templates/settings.html]: applied

File: auth.py [hunk 4]
```diff
@@ -14,6 +14,7 @@
 WOTT_SCREENLY_CREDENTIAL_NAME = 'screenly'
 
 
+
```
Comment (a-martynovich) [inline auth.py]: this causes our build to fail with PEP8 error

File: templates/settings.html [hunk 5]
```diff
@@ -187,7 +187,7 @@ <h1 class="page-header">
                     <div class="form-group row" id="curpassword_group">
                         <label class="col-2 col-form-label">Current Password</label>
                         <div class="col-3">
-                            <input class="form-control" name="curpassword" type="password" value="">
+                            <input class="form-control" name="current-password" type="password" value="">
```
Comment (a-martynovich) [inline templates/settings.html]: this should also be changed in `settings.py:settings_page()`

File: auth.py (hunk 4 above)
Comment (eugenosm) [reply auth.py]: fixed

File: templates/settings.html (hunk 5 above)
Comment (eugenosm) [reply templates/settings.html]: fixed

File: auth.py [hunk 6]
```diff
@@ -220,14 +220,7 @@ def _check(self, username, password):
     @property
     def template(self):
         return None
-
-
 def authorized(orig):
-    """
```
Comment (a-martynovich) [inline auth.py]: why was this removed?

File: docs/wott-authentication.md [hunk 7]
```diff
@@ -6,15 +6,15 @@ Access to Screenly web configuration interface can be restricted via an [*HTTP b
 ## Configuration
 
 After installing [*wott-agent*](https://www.wott.io/) and claiming the device in [*wott-dashboard*](https://dash.wott.io).
-Create credential with `name` `screenly_credentials` and `key` `login_credentials` contains colon separated username and password ( for example `pi:raspberry` ).
-In `owner` field enter linux user under which Screenly-ose ran on your device ( f.ex. `pi`) 
+Create credential with `name` `screenly` and `key` `login_credentials` contains colon separated username and password ( for example `pi:raspberry` ).
```
Comment (a-martynovich) [inline docs/wott-authentication.md]: key `login`

Comment (eugenosm) [reply docs/wott-authentication.md]: yeah....i miss that

File: auth.py (hunk 6 above)
Comment (eugenosm) [reply auth.py]: restored
